### ISSUE TITLE
It stops

### ISSUE THREAD

**Issue by Vipul006** [aa-miner/aaminer#25]
Heya,
It's very nice app.but after running it stops suddenly without any reason in my all three mobiles

**Comment by riansouza055** [comment]
Sim, com dificuldade ele para ainda em segundo plano então sugiro que usar o app com a tela do app fixa ou configure o smartphone para nunca descansar a tela

**Comment by Vipul006** [comment]
> Sim, com dificuldade ele para ainda em segundo plano então sugiro que usar o app com a tela do app fixa ou configure o smartphone para nunca descansar a tela

Thanks
